src/main/java/algorithms/curated170/medium/binarytreeverticalordertraversal/BinaryTreeVerticalOrderTraversalBFSTreeNode.java

```diff
@@ -0,0 +1,71 @@
+package algorithms.curated170.medium.braceexpansion;
+
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Deque;
+import java.util.List;
+import java.util.TreeMap;
+
+import algorithms.datastructures.TreeNode;
+
+public class BinaryTreeVerticalOrderTraversalBFSTreeNode {
```

Comment:
You mean **BinaryTreeVerticalOrderTraversalBFSTreeMap** I think.

Reply:
Yes.